frontend/src/components/eMIB/Email.jsx

```diff
@@ -1,20 +1,103 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
+import LOCALIZE from "../../text_resources";
+
+const styles = {
+  email: {
+    textAlign: "left",
+    paddingLeft: 24,
+    paddingTop: 24
+  },
+  replyStatus: {
+    fontSize: 16,
+    textAlign: "right",
+    fontWeight: "bold",
+    float: "right"
+  },
+  replyAndUser: {
+    color: "#00565E"
+  },
+  hr: {
+    width: "100%",
+    borderTop: "2px solid #00565E",
+    margin: "24px 0 24px 0"
+  },
+  capitalize: {
+    textTransform: "uppercase"
+  }
+};
 
 class Email extends Component {
   static propTypes = {
-    email: PropTypes.object.isRequired
+    email: PropTypes.object.isRequired,
+    respondToEmail: PropTypes.func.isRequired,
+    isRepliedTo: PropTypes.bool.isRequired
+  };
+
+  replyToEmail = () => {
+    this.props.respondToEmail(this.props.email.id);
+  };
+
+  addTaskToEmail = () => {
+    this.props.respondToEmail(this.props.email.id);
   };
 
   render() {
+    const email = this.props.email;
+
     return (
-      <span>
-        <div>{this.props.email.subject}</div>
-        <div>{this.props.email.from}</div>
-        <div>{this.props.email.to}</div>
-        <div>{this.props.email.date}</div>
-        <div>{this.props.email.body}</div>
-      </span>
+      <div style={styles.email}>
+        <div>
+          <h5 style={styles.capitalize}>
+            {LOCALIZE.emibTest.inboxPage.emailId}
```

Comment:
Instead of creating a style only to capitalize the text, you could just remove the style and do that:

`{LOCALIZE.emibTest.inboxPage.emailId.toUpperCase()}`

Reply:
fixed in 9d2402b